src/pages/resource/ReportResource.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, useState } from 'react';
+import ReportProblemIcon from '@material-ui/icons/ReportProblem';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { DialogActions, DialogContent, TextField, useMediaQuery } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import ErrorBanner from '../../components/ErrorBanner';
+import { useTranslation } from 'react-i18next';
+import { DeviceWidths } from '../../themes/mainTheme';
+import { reportResource } from '../../api/workListApi';
+import styled from 'styled-components';
+import { Resource } from '../../types/resource.types';
+
+const StyledButton = styled(Button)`
+  min-width: 10rem;
+`;
+
+interface ReportResourceProps {
+  resource: Resource;
+  setRequestSentSuccess: (success: boolean) => void;
+}
+
+const ReportResource: FC<ReportResourceProps> = ({ resource, setRequestSentSuccess }) => {
```

Comment:
Vi har slutta helt å bruke `FC` da det egentlig ikke var særlig mye hjelp i det. Den legger bl.a. til `children` som prop, men i mange komponenter vil vi ikke ha `children`, og i de tilfellene blir det egentlig bare mer feil å bruke `FC`
```suggestion
const ReportResource = ({ resource, setRequestSentSuccess }: ReportResourceProps) => {
```

Reply:
Noterer det til når Per kommer tilbake.